modules/identity/src/Volo.Abp.Identity.Blazor/Pages/Identity/RoleManagement.razor

```diff
@@ -28,7 +30,7 @@
           ShowPager="true"
           PageSize="PageSize">
     <DataGridColumns>
-        @if (ShouldShowEntityActions)
+        @if ( ShouldShowEntityActions )
```

Comment:
Again, we don't use extra spaces. Please change the editor settings. Otherwise, we see a lot of unnecessary changes whenever you touch a code file.

Reply:
Fixed